discord_bots/commands.py

```diff
@@ -1019,21 +1034,25 @@ async def del_(ctx: Context, *args):
             ),
             inline=True,
         )
+    for queue in queues_to_del_by_queue_waitlist_player:
+        session.query(QueueWaitlistPlayer).filter(
+            QueueWaitlistPlayer.queue_id == queue.id,
+            QueueWaitlistPlayer.player_id == message.author.id,
+        ).delete()
 
-    # TODO: Check deleting by name / ordinal
-    # session.query(QueueWaitlistPlayer).filter(
-    #     QueueWaitlistPlayer.player_id == message.author.id
-    # ).delete()
-
+    embed_description = ""
     if queues_to_del:
-        embed_description = f"<@{message.author.id}> removed from **{', '.join([queue.name for queue in queues_to_del])}**"
+        embed_description += f"**{message.author.display_name}** removed from **{', '.join([queue.name for queue in queues_to_del])}**"
+        embed.color = discord.Color.green()
+    if queues_to_del_by_queue_waitlist_player:
+        if embed_description:
+            embed_description += "\n"
+        embed_description += f"**{message.author.display_name}** removed from the waitlist for **{', '.join([queue.name for queue in queues_to_del_by_queue_waitlist_player])}**"
```

Comment:
alternatively the message could just be "removed from queues <names>" and not distinguish between waitlists and "proper" adds
reduces spam and is easier to grasp

Reply:
Fair point -- I thought the distinction would help make it clearer. Instead, I'll just merge the two lists together and print one message as suggested. Thanks!